confignode/src/main/java/org/apache/iotdb/confignode/conf/ConfigNodeConfig.java

```diff
@@ -101,6 +101,9 @@ public class ConfigNodeConfig {
   /** The least number of DataRegionGroup for each Database. */
   private volatile int leastDataRegionGroupNum = 5;
 
+  /** Indicate whether the leastDataRegionGroupNum is set by user, if true, lock it. */
+  private volatile boolean leastDataRegionGroupNumSetByUser = false;
```

Comment:
![image](https://user-images.githubusercontent.com/87789683/215939722-a04bad41-c90a-498e-a3c6-04129f6819dd.png)
This change should be cherry-picked too, otherwise the IT will fail.

Reply:
The codes in rel/1.0 is different from master. There is no `testLeastDataRegionGroupNum` variable in the test class.